.github/workflows/test.yml

```diff
@@ -35,7 +35,7 @@ jobs:
       - run: npm test
 
   build-mac:
-    runs-on: macOS-latest
+    runs-on: macos-latest
```

Comment:
I'm not 100% certain, but it looks like it shouldn't have because the values that used the `macOS` casing needed a specified version number (though have since been [deprecated](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#choosing-github-hosted-runners))